### PR TITLE
🐛 Docs button misalignment in sidebar

### LINKED ISSUE #719: [BUG] Docs button misalignment in sidebar.
**Describe the bug**
The docs button is misaligned in the sidebar bar.

**To Reproduce**
1. Open the streamETH studio app https://streameth.org/studio/

**Expected behavior**
The docs button should be properly aligned with the other buttons in the sidebar, maintaining consistent spacing and positioning.

**Screenshots**

CleanShot 2024-07-03 at 16 36 45

**Desktop (please complete the following information):**
 - Browser: chrome

### PR DESCRIPTION
**Pull request by Mario-SO**
# Pull Request Info

## Description

I moved the Docs icon to be with the rest of the icons so it is properly aligned with them. Previously, it wasn't aligned because the `SidebarMenu.tsx` component didn't accept external links. I've added the ability to handle external links in the sidebar menu.

Fixes (#719)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published

## Screenshots (if appropriate):

CleanShot 2024-07-04 at 11 40 38

CleanShot 2024-07-04 at 11 41 24

## Additional context:

closes #719

### CODE REVIEW

**Comment by xannyxs** [comment]
Can we add a small arrow on the Docs tab to show that it opens a new tab?

`import { LuArrowUpRight } from "react-icons/lu";`

File: packages/app/components/Sidebar/SidebarMenu.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 'use client'
 
+import { BookOpenText } from 'lucide-react'
```
Comment (xannyxs) [inline packages/app/components/Sidebar/SidebarMenu.tsx:3]: Please use react-icons/lu instead of lucide-react

Comment (Mario-SO) [reply packages/app/components/Sidebar/SidebarMenu.tsx:3]: sorry, didn't check that you are actually using the `lu` icons I'm used to `lucide` but it is true that they are going to get rid of it or something, i saw it somewhere hehe.

Comment (xannyxs) [reply packages/app/components/Sidebar/SidebarMenu.tsx:3]: `react-icons/lu` is `lucide`. The package name is just different

**Comment by Mario-SO** [comment]
> Can we add a small arrow on the Docs tab to show that it opens a new tab?
> 
> 
> 
> `import { LuArrowUpRight } from "react-icons/lu";`

It doesnt look that good... I think it is ok as it is. I will push the `lu` change later, i'm currently out

**Review by greatsamist — APPROVED** [review]

**Comment by pblvrt** [comment]
Not sure if adding an isExternal flag is the most elegant solution. Maybe it's just better that all sidebar item objects have a required attribute `url`. This would support both external and internal links and make the code cleaner imo @Mario-SO

**Comment by Mario-SO** [comment]
@pblvrt didn't want to mess up partners work, I agree having 1 url field is simpler.

**Pull request merged**